tests/pages/services/ServiceFormModal-cy.js

```diff
@@ -728,11 +728,6 @@ describe("Service Form Modal", function() {
           .contains("Add Placement Constraint")
           .click();
 
-        cy
```

Comment:
I think this is good removing this because the first field is now a select, which we don't need to focus. 👍